test/CMakeLists.txt

```diff
@@ -601,7 +630,9 @@ if(CELERITAS_USE_Geant4)
     ENVIRONMENT "${_geant4_test_env}")
   celeritas_add_test(accel/detail/HitProcessor.test.cc
     ENVIRONMENT "${_geant4_test_env}")
-  celeritas_add_test(accel/detail/TouchableUpdater.test.cc)
+  if(CELERITAS_REAL_TYPE STREQUAL "double")
+    celeritas_add_test(accel/detail/TouchableUpdater.test.cc)
+  endif()
```

Comment:
```suggestion
  celeritas_add_test(accel/detail/TouchableUpdater.test.cc ${_needs_double})
```

Reply:
It's different: compiling versus not running. I'll check again to see if this one fails to compile and if so will update the comment

Reply:
Yeah I forgot that this test uses `GenericGeoTestBase<GeantGeoParams>` which won't compile with single precision.